{
    "type": "bundle",
    "id": "bundle--d3e1484b-3968-4e0c-840b-9538f3897634",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--4b0212bd-f7b9-4489-b247-b0e6c6b6b095",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2023-05-10T00:05:37.576372Z",
            "modified": "2023-05-10T00:05:37.576372Z",
            "name": "CVE-2022-48382",
            "description": "In log service, there is a possible out of bounds write due to a missing bounds check. This could lead to local denial of service with System execution privileges needed.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2022-48382"
                }
            ]
        }
    ]
}